.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+	"env": {
+		"browser": true,
+		"es2021": true
+	},
+	"extends": ["eslint:recommended", ".prettierrc"],
+	"overrides": [
+		{
+		"files": [ "src/**/*.js" ],
+	}
+		],
+	"parserOptions": {
+		"ecmaVersion": "latest",
+		"sourceType": "module"
+	},
+	"rules": {
+		"no-useless-call": "error",
+        "no-useless-catch": "error",
+		"no-lonely-if": "error",
+		"no-duplicate-imports": "error",
+        "no-else-return": "error",
+        "no-empty-function": "error",
+		"eqeqeq": "error",
+		"curly": "error",
+		"no-var": "warn",
+		"indent": [
+			"error",
+			"tab"
+		],
+		"linebreak-style": [
+			"error",
+			"unix"
+		],
+		"quotes": [
+			"error",
+			"double"
+		],
+		"semi": "error",
+        "semi-spacing": "error",
+		"semi-style": [
+            "error",
+            "last"
+        ]
+	}
+}
```

Comment:
@Willian1661 

Could you add this rule please `eol-last`

Reply:
ok, i did 
i was reading about other rules, but i think for now we don't need much rules running through the code, don't we?

Reply:
i added these rules into the script: 
"no-multiple-empty-lines": "error",
"eol-last": [
			"error", 
			"always"
		],
it's all done here with the rules

Reply:
@Willian1661 

When you going to use block code 

Use
```